vertexai/snippets/src/main/java/vertexai/gemini/GroundingWithPublicData.java

```diff
@@ -43,15 +43,16 @@ public static String groundWithPublicData(String projectId, String location, Str
     // Initialize client that will be used to send requests.
     // This client only needs to be created once, and can be reused for multiple requests.
     try (VertexAI vertexAI = new VertexAI(projectId, location)) {
-      Tool googleSearchTool = Tool.newBuilder()
-          .setGoogleSearchRetrieval(
-              // Enable using the result from this tool in detecting grounding
-              GoogleSearchRetrieval.newBuilder().setDisableAttribution(false))
```

Comment:
Why are we turning off the attribution here?

Reply:
This is a preview attribution for testers, and the backend has removed this field.